public/javascripts/index.js

```diff
@@ -41,17 +28,39 @@ const Letter = React.createClass({
 	    </div>
 	)
     }
-})
+});
 
-const Alphabet = React.createClass({
-    render() {
+var CapitalAButton = React.createClass({
+    showCapitalA: function(){
+	ReactDOM.render(<Letter />, document.querySelector("#letter"))
+    },
+    
+    render: function() {
 	return (
-	    <div id="a">
-	    <ButtonChoices />
-	    <Letter />
+	    <div id="capital-button" onClick={this.showCapitalA}>
+	    A
 	    </div>
-	);
-    },
+	)
+    }
+
 });
 
-ReactDOM.render(<Alphabet />, document.querySelector('#container'));
+var ButtonChoices = React.createClass({
+    render: function() {
+	return (
+	    <div id="buttons">
+	    <div id="button-group">
+	    <CapitalAButton />
+	    </div>
+	    </div>
+	)
+    }
+});
+
+
+ReactDOM.render(
+    <div id="a">
+    <ButtonChoices />
+    </div>,
+    document.querySelector('#container')
+);
```

Comment:
thanks a lot for this feedback, @blairanderson. I think I implemented something along these lines in my latest commit which you can see [here](https://github.com/theonlyrao/alpha_animate/blob/master/public/javascripts/index.js#L33-L71).

As you will see, I still had to use a second `ReactDOM.render`. When I just tried to call the `<Letter />` component within the `<CapitalAButton />` React would naturally shove the whole letter into the actual button. But I need the letter to show up in a separate div.

React actually provided me with a very useful, if begrudging, error message indicating I could use the `componentDidUpdate` listener to update a component in a different part of the DOM.

In implementing this solution though, I get the feeling that it's a bit of a hack, and the right way would be to rethink the relationships on the page, and actually set things up properly and parents and children.